static/css/siglens.css

```diff
@@ -5364,4 +5364,8 @@ input.form-control {
     background: var(--component-bg-color);
     color: var(--component-text-color) !important;
     width: 100px;
+}
+
+#index-del-btn-traces{
+    background-image: none;
```

Comment:
Will this actually prevent the button from being clicked, or just make it look like there isn't a button there?